### ISSUE TITLE
How to generate t2_ft.txt and how to use it?

### ISSUE THREAD

**Issue by luomingshuang** [xiaomabufei/CAT#3]
Hi, @xiaomabufei , I have some questions about how to generate t2_ft.txt and how to use it.
 
According to my understanding, as the method shown in OWOD paper, the owod model will first be trained with t1_train.txt which includes 20 classes. This is the task 1. And then the model will be trained with t2_train.txt (which includes the new 20 classes) based on the pre-trained model which is trained with t1_train.txt. For alleviating forgetting, here use  exemplar replay. So the t2_ft.xt will include some instances which each class (20 classes in task 1 and 20 classes in task 2) contains  50 instances. We use the exemplar replay to finetune the trained task 2 model. 

Is my understanding correct? And I don't konw how to generate t2_ft.txt or what does the rule it base on?

**Comment by luomingshuang** [comment]
Em...thanks. But I think the tx_tf.txt for exemplar relay (using for alleviating forgetting) is not unique. Because when we choose 
 samples for finetune the model, we choose them randomly.

**Comment by luomingshuang** [comment]
Maybe we can see the codes in `https://github.com/JosephKJ/OWOD/blob/master/datasets/coco_utils/balanced_ft.py`. In the 64 line, it uses `random.shuffle(file_names)`.

**Comment by xiaomabufei** [comment]
Hello, thank you very much for your interest in our work, you are right, the ft data is really randomly selected.  But we did not make contributions for this issue in this work. Thus, we just used the same tx_ft.txt with existing works for fair comparison and convenience. In our furture works, we will also try to find better ways of playing back the data. If you are also interested in this issue, we can talk according to the E-mial.
